service/frontend/configs/quotas.go

```diff
@@ -106,6 +106,7 @@ var (
 		"/temporal.api.workflowservice.v1.WorkflowService/DescribeWorkflowExecution":     3,
 		"/temporal.api.workflowservice.v1.WorkflowService/DescribeTaskQueue":             3,
```

Comment:
what's the plan here? do we need to use a different rate limit here if it requests reachability (or always)?

Reply:
No DescribeTaskQueue will be subject to Visibility limit regardless of reachability requested or not, until we support content-based rate limiting.